scala-package/infer/src/test/java/org/apache/mxnet/infer/javaapi/ObjectDetectorOutputTest.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.mxnet.infer.javaapi;
+
+import org.junit.Test;
+
+public class ObjectDetectorOutputTest {
+
+    @Test
+    public void testConstructor() {
+
+        float[] arr = new float[]{0f, 1f, 2f, 3f, 4f};
+
+        ObjectDetectorOutput odOutput = new ObjectDetectorOutput("simba", arr);
+
+        assert (odOutput.getClassName().equals("simba"));
+        assert (odOutput.getProbability() == 0);
+        assert (odOutput.getXMin() == 1);
+        assert (odOutput.getXMax() == 2);
+        assert (odOutput.getYMin() == 3);
+        assert (odOutput.getYMax() == 4);
+    }
+
+    @Test (expected = ArrayIndexOutOfBoundsException.class)
+    public void testIncompleteArgsConstructor() {
+
+        float[] arr = new float[]{0f, 1f};
+
+        ObjectDetectorOutput odOutput = new ObjectDetectorOutput("simba", arr);
+
+        odOutput.getYMax();
```

Comment:
If you want to be pedantic, you could also check that probability is 0 and getXmin are also set as expected. :)

Reply:
Thanks. Done in 41f33716118c7cad36605daad820cc97a2e46201